guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -11,6 +11,7 @@ endif::[]
 ifdef::foreman-el,katello,satellite[]
 * xref:#repositories-rhel-8[Red Hat Enterprise Linux 8]
 * xref:#repositories-rhel-7[Red Hat Enterprise Linux 7]
+>>>>>>> da40753a (Add the Repositories and repair the content for RHEL 8)
```

Comment:
something went wrong with fixing conflicts...

Reply:
Yes. The Atom might be. Resolved.